src/functionalTest/java/uk/gov/hmcts/reform/em/hrs/IngestScenarios.java

```diff
@@ -105,7 +94,7 @@ public void shouldCreateHearingRecordingSegmentForVh() throws Exception {
 
         long vhFileSize = testUtil.getFileSizeFromStore(filenames, testUtil.vhBlobContainerClient);
         long hrsFileSize = testUtil.getFileSizeFromStore(filenames, testUtil.hrsVhBlobContainerClient);
-        Assert.assertEquals(hrsFileSize, vhFileSize);
+        Assertions.assertEquals(hrsFileSize, vhFileSize);
```

Comment:
Move all these Assertion import to static imports?

Reply:
fixed